### PR TITLE
Edits to PGD doc

### PR DESCRIPTION
**Pull request by ebgitelman**
Some were edited, some was new content, some had never been edited.

### CODE REVIEW

File: product_docs/docs/pgd/5/other_considerations.mdx [hunk 1]
```diff
@@ -4,33 +4,33 @@ title: "Other considerations"
...
 
-## Data Consistency
+## Data consistency
 
 Read about [Conflicts](consistency/conflicts/) to understand the implications of the asynchronous operation mode in terms of data consistency.
 
 ## Deployment
 
-EDB PGD is intended to be deployed in one of a small number of known-good configurations, using either [Trusted Postgres Architect](/tpa/latest) or a configuration management approach and deployment architecture approved by Technical Support.
+PGD is intended to be deployed in one of a small number of known-good configurations, using either [Trusted Postgres Architect](/tpa/latest) or a configuration management approach and deployment architecture approved by Technical Support.
```
Comment (djw-m) [inline product_docs/docs/pgd/5/other_considerations.mdx:13]: EDB PGD - it's a product reference.

File: product_docs/docs/pgd/5/postgres-configuration.mdx [hunk 2]
```diff
@@ -21,31 +21,31 @@ PGD requires these Postgres settings to run correctly:
...
+-   `max_replication_slots` &mdash; Two needed for every peer node.
 -   `wal_sender_timeout` and `wal_receiver_timeout` &mdash; Determines how
     quickly a node considers its CAMO partner as disconnected or
     reconnected. See [CAMO failure scenarios](durability/camo/#failure-scenarios) for
     details.
 
 In normal running for a group with N peer nodes, PGD requires
 N slots and WAL senders. During synchronization, PGD temporarily uses another
-N - 1 slots and WAL senders, so be careful to set the parameters high enough
+N -1 slots and WAL senders, so be careful to set the parameters high enough
```
Comment (djw-m) [inline product_docs/docs/pgd/5/postgres-configuration.mdx:48]:
```suggestion
N-1 slots and WAL senders, so be careful to set the parameters high enough
```

File: product_docs/docs/pgd/5/postgres-configuration.mdx [hunk 3]
```diff
@@ -78,4 +76,4 @@ timestamp to use.
 
 ### `max_prepared_transactions`
 
-Needs to be set high enough to cope with the maximum number of concurrent prepared transactions across the cluster due to explicit two-phase commits, CAMO, or Eager transactions. Exceeding the limit prevents a node from running a local two-phase commit or CAMO transaction and prevents all Eager transactions on the cluster. You can set this only at Postgres server start.
+Needs to be set high enough to cope with the maximum number of concurrent prepared transactions across the cluster due to explicit two-phase commits, CAMO, or Eager transactions. Exceeding the limit prevents a node from running a local two-phase commit or CAMO transaction and prevents all Eager transactions on the cluster. You can set this parameter only at Postgres server start.
```
Comment (djw-m) [inline product_docs/docs/pgd/5/postgres-configuration.mdx:79]:
```suggestion
Needs to be set high enough to cope with the maximum number of concurrent prepared transactions across the cluster due to explicit two-phase commits, CAMO, or Eager transactions. Exceeding the limit prevents a node from running a local two-phase commit or CAMO transaction and prevents all Eager transactions on the cluster. You can only set this parameter at Postgres server start.
```

**Review by djw-m — CHANGES_REQUESTED** [review]
Couple of small tweaks.

This review has also raised up some tangential issues with the known issues content. That's going to be reviewed and revised.

File: product_docs/docs/pgd/5/other_considerations.mdx (hunk 1 above)
Comment (drothery-edb) [reply product_docs/docs/pgd/5/other_considerations.mdx:13]: > EDB PGD - it's a product reference.

Right, we don't need EDB when using the short name

File: product_docs/docs/pgd/5/postgres-configuration.mdx (hunk 3 above)
Comment (ebgitelman) [reply product_docs/docs/pgd/5/postgres-configuration.mdx:79]: Hi. This modifier as placed makes it ambiguous. Is it only this parameter you can set at Postgres server start? Or is the only time you can set it at server start? If the latter (which it appears to be based on the original text), my edit is correct.

Comment (djw-m) [reply product_docs/docs/pgd/5/postgres-configuration.mdx:79]:
```suggestion
Needs to be set high enough to cope with the maximum number of concurrent prepared transactions across the cluster due to explicit two-phase commits, CAMO, or Eager transactions. Exceeding the limit prevents a node from running a local two-phase commit or CAMO transaction and prevents all Eager transactions on the cluster. This parameter can only be set at Postgres server start.
```

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**